### PR TITLE
Lua metadata

### PR DESCRIPTION
**Pull request by kukulam-allegro**

### CODE REVIEW

File: envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EnvoySnapshotFactoryTest.kt [hunk 1]
```diff
@@ -45,6 +45,7 @@ class EnvoySnapshotFactoryTest {
         const val EGRESS_PORT = 3380
         const val CLUSTER_NAME = "cluster-name"
         const val DEFAULT_SERVICE_NAME = "service-name"
+        const val DEFAULT_DISCOVER_SERVICE_NAME = "service-name"
```
Comment (malfoj89) [inline envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EnvoySnapshotFactoryTest.kt:48]: I'd rather make it unique or reuse DEFAULT_SERVICE_NAME 
```suggestion
        const val DEFAULT_DISCOVER_SERVICE_NAME = "discovery-service-name"
```

**Review by KSmigielski — APPROVED** [review]

File: envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EnvoySnapshotFactoryTest.kt (hunk 1 above)
Comment (konrad-kaminski) [reply envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EnvoySnapshotFactoryTest.kt:48]: Also change the constant name to `DEFAULT_DISCOVERY_SERVICE_NAME`.

File: envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/TestNodeFactory.kt [hunk 2]
```diff
@@ -27,6 +28,10 @@ fun nodeV3(
         meta.putFields("service_name", string(serviceName))
     }
 
+    discoveryServiceName?.let {
+        meta.putFields("service_name", string(discoveryServiceName))
```
Comment (konrad-kaminski) [inline envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/TestNodeFactory.kt:32]: Shouldn't it be `"discovery_service_name"`?

**Review by konrad-kaminski — CHANGES_REQUESTED** [review]

File: envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/TestNodeFactory.kt (hunk 2 above)
Comment (kukulam-allegro) [reply envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/TestNodeFactory.kt]: done

File: envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EnvoySnapshotFactoryTest.kt (hunk 1 above)
Comment (kukulam-allegro) [reply envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EnvoySnapshotFactoryTest.kt]: done

**Review by malfoj89 — APPROVED** [review]

**Review by konrad-kaminski — APPROVED** [review]

**Review by Automaat — APPROVED** [review]

**Pull request merged**